misc/userscripts/password_fill

```diff
@@ -236,7 +236,7 @@ pass_backend() {
         if ((match_line)) ; then
             # add entries with matching URL-tag
             while read -r -d "" passfile ; do
-                if $GPG "${GPG_OPTS}" -d "$passfile" \
+                if $GPG "${GPG_OPTS[@]}" -d "$passfile" \
```

Comment:
ping @t-wissmann - was the intent here to print all `GPG_OPTS` or just the quite flag or something? This change was for this warning:

> SC2128: Expanding an array without an index only gives the first element.